server/komodo-rest/src/main/java/org/komodo/rest/KomodoService.java

```diff
@@ -278,13 +278,23 @@ protected Response createErrorResponse(Status returnCode, List<MediaType> mediaT
         else
             resultMsg = RelationalMessages.getString(errorType, errorMsgInputs, buf.toString());
 
-        Object responseEntity = createErrorResponseEntity(mediaTypes, resultMsg);
-        return Response.status(returnCode).entity(responseEntity).build();
+        return createErrorResponse(returnCode, mediaTypes, resultMsg);
     }
+    
+    protected Response createErrorResponse(Status returnCode, List<MediaType> mediaTypes,
+                                           RelationalMessages.Error errorType, Object... errorMsgInputs) {
+        String resultMsg = null;
+        if (errorMsgInputs == null || errorMsgInputs.length == 0)
+            resultMsg = RelationalMessages.getString(errorType);
+        else
+            resultMsg = RelationalMessages.getString(errorType, errorMsgInputs);
 
-    protected Response createErrorResponse(List<MediaType> mediaTypes, Throwable ex,
-                                                                       RelationalMessages.Error errorType, Object... errorMsgInputs) {
```

Comment:
This is a product of the discussion below concering Status.FORBIDDEN being added to all createErrorResponse methods.